format/title_page.tex

```diff
@@ -33,6 +33,12 @@
 	{\large{Candidato:}{\normalsize\vspace{3mm} \bf\\ \large{Sara Durante}}}
 \end{minipage}
 
+\vspace{10mm}
+
+\begin{center}
+    {\large{Anno Accademico 2021-2022}}
+\end{center}
+
```

Comment:
wow, you solve my problem :) I was procrastinating to find a solution to this, thanks ❤️